R/install-dev.sh

```diff
@@ -35,12 +35,19 @@ LIB_DIR="$FWDIR/lib"
 mkdir -p $LIB_DIR
 
 pushd $FWDIR > /dev/null
+if [ ! -z "$1" ]
+  then
+    R_HOME="$1/bin"
+   else
+    R_HOME="$(dirname $(which R))"
```

Comment:
`R_HOME` is a reasonably well defined concept ? http://www.inside-r.org/r-doc/base/R.home 

Regardless I think you will need to use a different R binary for running sparkR if you don't want it to use the default R ?